atlasexec/atlas.go

```diff
@@ -335,8 +335,9 @@ func (c *Client) SchemaInspect(ctx context.Context, params *SchemaInspectParams)
 	if params.DevURL != "" {
 		args = append(args, "--dev-url", params.DevURL)
 	}
-	if params.Format == "sql" {
-		args = append(args, "--format", "{{ sql . }}")
+	if params.Format != "" {
+		format := fmt.Sprintf("{{ %s . }}", params.Format)
+		args = append(args, "--format", format)
 	}
```

Comment:
```suggestion
	switch {
	case params.Format == "sql": 
		args = append(args, "--format", "{{ sql . }}")
	case params.Format != "":
		args = append(args, "--format", params.Format)
	}
```
I guess the `"sql"` wrapping is an old behavior, because we prefer passing the args to atlas as is.

Reply:
Unfortunately I tried this with a recent atlas installation and it didn't work, the format was just printed back to me
```
> atlas schema inspect -u "..." --format json
json
>
```

Reply:
Oh I see what you mean, thought you were suggesting the other way, that atlas will know to handle `--format json`